Seminars/Sem.01/Pract.01/Task07.cpp

```diff
@@ -0,0 +1,110 @@
+//Gabriela Mladenova
+//SI
+//Group 1
+//FN: 3MI0600225
+#include <iostream>
+
+const int SIZEOFARRAYS = 30;
+enum MovieGenre
+{
+    Comedy,
+    Action,
+    Horror,
+    Romantic
+};
+struct Movie
+{
+    char movieName[SIZEOFARRAYS + 1];
+    int duration;
+    MovieGenre movieGenre;
+    double rating;
+};
+void printMovies(const Movie movies[], const int SIZE)
+{
+    for (int i = 0; i < SIZE; ++i)
+    {
+        std::cout << "Movie name: " << movies[i].movieName << std::endl;
+        std::cout << "Duration of the movie: " << movies[i].duration << std::endl;
+        std::cout << "Movie genre: ";
+
+        if (MovieGenre::Comedy)
+        {
+            std::cout << "Comedy" << std::endl;
+        }
+        else if (MovieGenre::Action)
+        {
+            std::cout << "Action" << std::endl;
+        }
+        else if (MovieGenre::Horror)
+        {
+            std::cout << "Horror" << std::endl;
+        }
+        else if (MovieGenre::Romantic)
+        {
+            std::cout << "Romantic" << std::endl;
+        }
+        else
+        {
+            std::cout << "No genre" << std::endl;
+        }
+
+        std::cout << "Movie rating: " << movies[i].rating << std::endl;
+        std::cout << std::endl;
+    }
+}
+void PrintTopMovie(const Movie movies[], const int SIZE)
+{
+    int topMovieIndex = 0;
+    int topRating = 0;
+    for (int i = 0; i < SIZE; ++i)
+    {
+        if (movies[i].rating > topRating)
+        {
+            topRating = movies[i].rating;
+            topMovieIndex = i;
+        }
+    }
+    for (int j = 0; j < SIZE; ++j)
+    {
+        if (j == topMovieIndex)
+        {
+            std::cout << movies[j].movieName << " " << movies[j].duration
+                << " " << movies[j].movieGenre << " " << movies[j].rating;
+        }
+    }
```

Comment:
Just write cout<<movies[topMovieIndex].movieName<<" "<<movies[topMovieIndex].duration and so on.
This for loop is useless